templates/donation/informations.html.twig

```diff
@@ -65,7 +65,12 @@
         <div>
             {{ form_errors(form.address) }}
 
-            <div class="form__row address-autocomplete"></div>
+            <div class="form__row">
+                <div class="address-autocomplete"></div>
+                <p class="text--smallest text--muted text--left visually-hidden" id="address-autocomplete-help-message">
```

Comment:
If I force displaying this field client side, can I click on it (though bypassing the autocomplete)?

Reply:
yes, you can do it, as you can disable JS and bypass the autocomplete, or inter unknown address and this message will appear automatically

Reply:
Ok!